Test/Utils/MultiNode/ContrailMultiNodeProvisioning.ps1

```diff
@@ -15,7 +17,21 @@ function New-MultiNodeSetup {
         [Parameter(Mandatory = $true)] [String] $ContrailProject
     )
 
-    $ContrailRestApi = [ContrailRestApi]::new($ControllerConfig, $OpenStackConfig)
+    if ($ControllerConfig.AuthMethod -eq 'keystone') {
```

Comment:
I wonder if we could move this conditional branch to some method/function to separate logic.